src/test/java/com/galvanize/customer/controllers/CustomerRestControllerTests.java

```diff
@@ -28,41 +36,92 @@
     @Autowired
     CustomerRepository repository;
 
+    Customer c;
+
     @Before
     public void setUp() throws Exception {
-
+        c = new Customer();
+        c.setCustomerId(312L);
+        c.setFirstName("Jim");
+        c.setLastName("Jones");
+        c.setAddress("111 Main St");
+        c.setCity("Dallas");
+        c.setState("TX");
+        c.setZip("90210");
+        c.setPhoneNumber("111-222-3333");
```

Comment:
Missing a step here.  Should have repository.save(c) to commit the row to the db.